shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -419,9 +419,8 @@
 }
 
 SkCanvas* FlutterPlatformViewsController::CompositeEmbeddedView(int view_id) {
-  FML_DCHECK(flutter_view_);
-  // TODO(amirh): assert that this is running on the platform thread once we support the iOS
-  // embedded views thread configuration.
+  // Any UIKit related code has to run on main thread.
+  FML_DCHECK([[NSThread currentThread] isMainThread]);
```

Comment:
qq: what is enforcing that running on the main thread also means that `flutter_view_` is set?

Reply:
oh, my mistake, fixed.